scipy/io/mmio.py

```diff
@@ -505,7 +532,7 @@ def _parse_body(self, stream):
         return a
 
     #---------------------------------------------------------------------------
-    def _write(self, stream, a, comment='', field=None, precision=None):
+    def _write(self, stream, a, comment='', field=None, precision=None, symmetry=None):
```

Comment:
Yes, you are right. If the behaviour should be the same as before, `None` must remain the default value for `symmetry` and therefore the symmetry check would be enabled by default if no `symmetry` is passed.